generators/generator-base.js

```diff
@@ -2606,7 +2607,7 @@ templates: ${JSON.stringify(existingTemplates, null, 2)}`;
         return { ...defaultAppTypeConfig, ...defaultConfig };
     }
 
-    setConfigDefaults(defaults = defaultConfig) {
+    setConfigDefaults(defaults = this.jhipsterConfig.applicationType !== 'microservice' ? defaultConfig : defaultConfigMicroservice) {
```

Comment:
const { MONOLITH, MICROSERVICE, GATEWAY, UAA } = require('./application-types');

this.jhipsterConfig.applicationType !== MICROSERVICE

Reply:
yes you're right, I update that